assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -5733,10 +5733,9 @@ private String build_instructor_view_students_assignment_context(VelocityPortlet
                 "&searchFilterOnly=" + searchFilterOnly.toString() +
                 "&estimate=true";
         context.put("accessPointUrl", accessPointUrl);
-                           
-        Collection<Assignment> assignments = assignmentService.getAssignmentsForContext(contextString)
-                .stream().filter(a -> !assignmentService.assignmentUsesAnonymousGrading(a))
-                .collect(Collectors.toList()); 
+        Collection<Assignment> assignments = assignmentService.getAssignmentsForContext(contextString).stream()
+            .filter(Predicate.not(assignmentService::assignmentUsesAnonymousGrading))
+            .collect(Collectors.toSet()); 
```

Comment:
That's probably coming from my suggestion, but could you add one indentation here please?